samples/MusicStore/ForTesting/Mocks/StartupSocialTesting.cs

```diff
@@ -59,13 +61,13 @@ public void ConfigureServices(IServiceCollection services)
             }
 
             // Add Identity services to the services container
-            services.AddIdentity<ApplicationUser, IdentityRole>(options =>
-                    {
-                        options.Cookies.ApplicationCookie.AccessDeniedPath = new PathString("/Home/AccessDenied");
-                    })
+            services.AddIdentity<ApplicationUser, IdentityRole>()
                     .AddEntityFrameworkStores<MusicStoreContext>()
                     .AddDefaultTokenProviders();
 
+            services.Configure<CookieAuthenticationOptions>(IdentityCookieOptions.ApplicationScheme, 
```

Comment:
Is this separation going to cause discover-ability problems? Should you add Identity extensions for `services.Configure<CookieAuthenticationOptions>(IdentityCookieOptions.ApplicationScheme,`?